subset/dot1x/Dockerfile.test_dot1x

```diff
@@ -0,0 +1,13 @@
+FROM daqf/aardvark:latest
+
+RUN apt-get update && \
+    apt-get install -y freeradius iproute2 vim git gcc python3-dev musl-dev g++ tcpdump
```

Comment:
can you take a look at aardvark image and remove any duplicates. Not the end of the world, but might as well do one optimization path.

Reply:
Changed